test/functional/university_controller_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'test_helper'
+
+class UniversityControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers
+
+  test 'when picking a repo' do
+    get :show, params: { id: 'picking_a_repo' }
+
+    assert_response :success, 'successfully renders show'
+    assert_not_nil assigns(:page_title), 'assigns page title'
```

Comment:
Usage of `assigns` and `assert_template` is not encouraged Rails 5 onwards. So let's not use it.